ksql-engine/src/main/java/io/confluent/ksql/util/CleanUpUtil.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class CleanUpUtil {
```

Comment:
I don't see where this class is used and there are no tests for it

Reply:
I will use it in the clean up task.

Reply:
code shouldn't be added without tests